P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -1259,6 +1299,8 @@ exemplifying here too? -->
       </div>
       <div type="div4" xml:id="NDPERSEpe">
         <head>Personal Events</head>
+        <p>This section describes <gi>event</gi>s that relate to one or more <gi>person</gi>s, while the section on
+          <ref target="#NDEVNT">Events</ref> below describes the use of <gi>event</gi> in more general terms.</p>
         <p>Events in a person's history are not characteristics of an individual, but often cause an individual to gain such
           characteristics, or to enter a new state. Most such events, for example marriage, appointment, promotion, or a journey
           may be recorded using the generic element <gi>event</gi>, which may be grouped with <gi>listEvent</gi>, and has a
```

Comment:
Not here but in line 1309 (I couldn't comment there as this section wasn't changed): "in a person's life" with apostrophe instead of "persons life".
Perhaps you can sneak this correction in while working on this passage.